src/utils/mesh.js

```diff
@@ -261,14 +349,21 @@ export async function createMesh (runAIOCommand, installedPlugins) {
       paas,
       catalog,
       githubOrg,
-      githubRepo
+      githubRepo,
+      apiKey,
+      environmentId
     )
 
     const { meshUrl } = await runAIOCommand('api-mesh:create', [
       meshConfigFilePath,
       '-c'
     ])
-    // TODO: status call, when true, continue else retry if error
+
+    await new Promise((resolve) =>
```

Comment:
Do you need this initial timeout await?  shouldn't the resolution of the prior `create` command indicate it is ready to check?

Reply:
Unfortunately no, we need this initial timeout because mesh provisioning is async. Typically it takes around 30-45 seconds. What you see after create is just a notification saying your create request has been submitted. Hence we wait before we check the status and retry.

Reply:
Hm. So we have a forced 60s delay during the CLI process? I don't know if I'm a fan...

I'd like to ensure this doesn't block the user from continuing with the CLI. It should asynchronously perform this check/provision. If we can at least get the mesh url, that's all we need. Then the cli script can continue with writing other things and async the checkAndRetry can execute and warn/alert if unable to provision after all retries.

Reply:
Update it to be a non blocking process